SFR/common_input.i

```diff
@@ -8,6 +8,7 @@ edge_length = 12.1705  # side length of the fuel assembly (cm)
 lattice_pitch = 1.24  # Pitch used in lattice or mesh generation (cm)
 
 NUM_SECTORS              = 6  # Number of angular sectors in radial direction
```

Comment:
This name/comment are a little confusing since this appears to be used for `num_sides`?